### ISSUE TITLE
QuickFolders Options: tab of selected settings page looks same as all other tabs.

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [RealRaven2000/QuickFolders#379]
The screenshot below shows the options window when the 'bling' tab is selected.

![image](https://github.com/RealRaven2000/QuickFolders/assets/47056543/b65e29ff-7f6a-4629-b4f7-5515ef188abc)

The selected tab is identical visually to the other tabs (and that occurs whichever tab I select).

**My system**

QF 6.1pre28
Thunderbird 115.0.1 (x64)
Windows 10 Pro - 22H2 (2009: 19045)

**Comment by RealRaven2000** [comment]
that's interesting, are you using the default theme or dark theme or some other theme?

Here (also on Windows 10 Pro 22H2 with the dark theme selected), it looks like this (with dark theme):

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/a1b98cc5-3dcc-447e-ae3d-98fc45cff568)

I am using system color variables from the Thunderbird theming system itself. Note that your icons should be white also, not black...

can you try restarting Thunderbird to see if it makes a difference? When loading new Add-on versions, style sheets sometimes get messed up; that's partly due to the fact that it caches style sheets, similar to Firefox.

**Comment by LinuxOnTheDesktop** [comment]
It happens both when I use the 'auto' theme (and I have a Windows dark mode enabled) and when I use Tbird's 'dark' theme.

Restarting Tbird did not help.

I do have a custom UserChrome file, but I tried commenting out all of its contents (and restarting Tbird) and that did not solve the problem.

I tried disabling all addOns except QF. That did not help either, unfortunately.

**Comment by RealRaven2000** [comment]
@inuxOnTheDesktop 
Can you check whether this item is fixed? It looks good to me in both themes both in Thunderbird 102 and 115:
![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/4671f5ac-778e-4916-9408-2fba3afb52e9)

I made a lot of changes to the layout hoping to close out a few old ones with this version.

**Comment by LinuxOnTheDesktop** [comment]
I'm afraid the problem persists - in Thunderbird 115.1, using QF 6.1.pre28, on Windows.

**Comment by RealRaven2000** [comment]
Ok, I should have probably waited until 6.1 was reviewed, it is currently await that ATN to be pushed to the public. I am at revision 124 :)

[QuickFolders-mx-6.1pre123.zip](https://github.com/RealRaven2000/QuickFolders/files/12294785/QuickFolders-mx-6.1pre123.zip)

that is the last prerelease before I built 6.1. pre28 was built  on the 31/07 and I have been working on it every day. so there is a chance that it's different now. If you could load pre123 and make a screenshot that would be nice :)

**Comment by LinuxOnTheDesktop** [comment]
Here is a screenshot of version 'pre123'. The problem persists into that version, unfortunately.

![image](https://github.com/RealRaven2000/QuickFolders/assets/47056543/50a4d124-0b9a-45f8-a4d8-4a90556ad678)

**Comment by RealRaven2000** [comment]
the background color one is definitely strange... the icons could maybe be fixed by adding this setting in Settings / Advanced settings (about:config)
`svg.context-properties.content.enabled = true`

this is expected to fix coloring of icons only...

**Comment by LinuxOnTheDesktop** [comment]
I tried that about:config setting, and restarted, and saw no difference.

**Comment by RealRaven2000** [comment]
Is the problem still there?

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/1c998212-4030-4e72-86f4-8ecb12596c6c)

**Comment by LinuxOnTheDesktop** [comment]
I am afraid that the problem persists - on Thunderbird 115.4.1, with QF 6.3, on Linux Mint and on Windows.

**Comment by RealRaven2000** [comment]
can we close this one?

**Comment by LinuxOnTheDesktop** [comment]
Yes, closing seems appropriate, because so far as I can see the problem is fixed. Thank you.
